packages/pwa-kit-react-sdk/src/ssr/universal/utils.ts

```diff
@@ -32,9 +39,33 @@ export const getAssetUrl = (path: string) => {
         ? // @ts-ignore
           `${window.Progressive.buildOrigin as string}`
         : `${bundleBasePath}/${process.env.BUNDLE_ID || 'development'}/`
+
     return path ? `${publicPath}${path}` : publicPath
 }
 
+// TODO: Once we establish that we have a new @salesforce/pwa-kit-extensibility package, we can move this utility to
+// it as to not have direct references to extensibilty in the sdk. This will also reduce duplicate code.
+/**
+ * Get the URL that should be used to load a static asset from the bundle.
+ *
+ * @param {string} path - relative path from the build directory to the asset
+ * @function
+ * @returns {string}
```

Comment:
You may want to document the `opts.extensionPackageName` parameter, which optionally defines the extension package name for the asset.

Reply:
Makes sense. I've updated it. I also changed `extensionPackageName` to `appExtensionPackageName` because I should follow my own advice :P